__tests__/app.test.js

```diff
@@ -58,13 +58,37 @@ describe('App Tests', () => {
                 expect(article).toHaveProperty('author', 'butter_bridge');
                 expect(article).toHaveProperty('title', "Living in the shadow of a great man");
                 expect(article).toHaveProperty('body', "I find this existence challenging");
-                expect(article).toHaveProperty('topic', "mitch")
+                expect(article).toHaveProperty('topic', "mitch");
                 expect(article).toHaveProperty('created_at', '2020-07-09T20:11:00.000Z');
                 expect(article).toHaveProperty('votes', 100);
                 expect(article).toHaveProperty('article_img_url', "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700");
             })
         })
-
+    })
+    describe('GET `/api/articles` tests', () => {
+        test('200: returns status code 200 upon successful GET request', () => {
+            return request(app)
+            .get('/api/articles')
+            .expect(200);
+        })
+        test('200: returns all articles data with certain properties upon successful request', () => {
+            return request(app)
+            .get('/api/articles')
+            .then(({body: {articles}}) => {
+                expect(articles).toBeSorted('created_at', {descending:true})
+                
```

Comment:
Looking at the docs for `jest-sorted` I would have thought it would need to be the matcher _.toBeSortedBy_ as that's the alias for sorting an array of objects 
Maybe just double check this test is working